engine/class_modules/sc_death_knight.cpp

```diff
@@ -9066,9 +9060,11 @@ void death_knight_t::default_apl_blood()
   standard -> add_talent( this, "Bonestorm", "if=runic_power>=100&!buff.dancing_rune_weapon.up" );
   standard -> add_action( this, "Death Strike", "if=runic_power.deficit<=(15+buff.dancing_rune_weapon.up*5+spell_targets.heart_strike*talent.heartbreaker.enabled*2)|target.1.time_to_die<10" );
   standard -> add_action( this, "Death and Decay", "if=spell_targets.death_and_decay>=3" );
+  standard -> add_action( "deaths_due,if=spell_targets.death_and_decay>=3" );
   standard -> add_action( this, "Heart Strike", "if=buff.dancing_rune_weapon.up|rune.time_to_4<gcd" );
   standard -> add_action( this, "Blood Boil", "if=buff.dancing_rune_weapon.up" );
   standard -> add_action( this, "Death and Decay", "if=buff.crimson_scourge.up|talent.rapid_decomposition.enabled|spell_targets.death_and_decay>=2" );
+  standard -> add_action( "deaths_due,if=buff.crimson_scourge.up|talent.rapid_decomposition.enabled|spell_targets.death_and_decay>=2" );
```

Comment:
You could merge the old death and decay actions and the new death's due actions if you just plan to copy the conditions.

Reply:
I'll use any_dnd on these ones for now, but we may want to split them in the future when we have time to check into apl usage of them more.  It will come down to if there is more value in maintaining 100% uptime of the buff or not.  

Simply removing the target check for death's due was a slight (10) dps loss.
```
5 Target Patch
deaths_due                   : start=  5.3 refresh=291.9 interval= 58.4 trigger=  1.0 duration= 51.8 uptime= 93.35%
1 Target Patch - original apl
deaths_due                   : start=  8.1 refresh= 40.6 interval= 37.2 trigger=  5.8 duration= 27.2 uptime= 73.59%
1 Target Patch - no num_targets check
deaths_due                   : start=  5.9 refresh= 56.5 interval= 53.1 trigger=  4.8 duration= 47.2 uptime= 93.00%
```